modules/assets.test.js

```diff
@@ -5,7 +5,7 @@ const expect = chai.expect;
 
 const { getCachebustedPath, getCachebustedRealPath, getImagePath } = require('./assets');
 
-describe.only('assets', () => {
```

Comment:
I should stop using `.only` to debug tests…who knows how long this has been the only server test running?? 😓